essential_reading/DataSources/Data_S3.md

```diff
@@ -0,0 +1,223 @@
+# Data / Cloud Storage
+
+One of the main storage locations for HyTest data is in '_The Cloud_'. This is sometimes referred to as **'Object Storage'**.
+The data is kept in data centers operated by Amazon, Microsoft, or similar, which makes it easily available to network-connected devices.
+The main advantage of doing this is that if your compute engine is also in that same data center (or nearby, as is the case with many JupyterHub services),
+the data doesn't have to go very far to get to the compute power.
+This brings the computation to the data, rather than shipping large datasets across the internet to get to the compute engine.
+
+[S3](https://aws.amazon.com/s3/) is Amazon's implementation of object storage, which pairs with the Amazon (AWS) nodes on which the JupyterHub runs.
+What follows is a brief demo of how S3 data is accessed (both read and write), and some pitfalls to watch out for.
```

Comment:
```suggestion
The HyTEST project purchased a 1 PB [Open Storage Network (OSN)](https://www.openstoragenetwork.org/) pod, providing [Ceph Object Storage](https://docs.ceph.com/en/pacific/glossary/#term-Ceph-Object-Storage) on a high-speed network. Because we own this piece of hardware, we can host data without the storage or egress fees that come along with other forms of cloud object storage, such as [Amazon S3 object storage](https://aws.amazon.com/s3/).

Ceph object storage supports an [API that is compatible with the basic data access model of the Amazon S3 API](https://docs.ceph.com/en/pacific/radosgw/s3/#).

What follows is a brief demo of how data can be read from or written to the OSN pod, and some pitfalls to watch out for.
```

Reply:
@rsignell-usgs - would you like you to take a glance at what I drafted here